regulations/static/regulations/js/collapse.js

```diff
@@ -0,0 +1,14 @@
+function makeCollapsable(el) {
+    const collapse_buttons = el.querySelectorAll('.js-collapse');
+    for (const collapse_button of collapse_buttons) {
+        collapse_button.addEventListener('click', function() {
+            el.classList.toggle("collapsed");
```

Comment:
could use `data-state` or similar. The benefit is the more direct description of the element's state.